### PR TITLE
#2840 Post explore unify fixes

### LINKED ISSUE #2840: Can we have collection first in explore?
I feel like it's in reverse

+ I would like to see the tabs in Sentence case (first letter capitalised) 
+ plural "Collection" since it show multiple collections
+ missing "Showing result of ..." in `/explore/?tab=COLLECTION`
+ missing sort in collections tab

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2840

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot 2022-04-13 at 4 10 58 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
- Navbar Search is not vertical centered (I would remove useless card class)
- mobile searchbar is breaking
- I would change color (greyer) of search term `{{ $route.query.search }}` (could be fun to use `text__stroked` class) and why not increase its size
- I would remove `is-small` from tabs to have something consistent
- can we have Collections first? 🥺

**Comment by prachi00** [comment]
@roiLeo which mobile search?

**Comment by roiLeo** [comment]
> @roiLeo which mobile search?

this one 
Capture d’écran 2022-04-13 à 1 33 00 PM

**Comment by prachi00** [comment]
Also should we have collections first? Earlier we discussed it should be gallery then collection then we decided on gallery again  (because of empty collections thing) 😂

**Comment by prachi00** [comment]
@roiLeo added collection at first for now, did the other fixes mentioned as well

**Comment by prachi00** [comment]
Screenshot 2022-04-13 at 5 29 43 PM

**Comment by roiLeo** [comment]
should it go before or after #2705?

**Comment by prachi00** [comment]
> should it go before or after #2705?

isnt this already merged?

**Comment by roiLeo** [comment]
> > should it go before or after #2705?
> 
> isnt this already merged?

Oupsi was refering to #2718

File: components/rmrk/ExploreLayout/ExploreLayout.vue [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
   <div>
-    <b-tabs v-model="selectedTab" size="is-small">
-      <b-tab-item label="GALLERY" value="GALLERY"><Gallery /></b-tab-item>
-      <b-tab-item label="COLLECTION" value="COLLECTION">
+    <div v-if="$route.query.search" class="mb-3 is-flex is-align-items-center">
+      Showing results for
+      <span class="text__stroked search-term ml-3">{{
+        $route.query.search
+      }}</span>
+    </div>
```
Comment (roiLeo) [inline components/rmrk/ExploreLayout/ExploreLayout.vue:8]: I would change this by
```suggestion
    <div v-if="$route.query.search" class="block">
      Showing results for
      <span class="text__stroked is-size-3">{{
        $route.query.search
      }}</span>
    </div>
```

File: components/rmrk/ExploreLayout/ExploreLayout.vue [hunk 2]
```diff
@@ -48,4 +54,8 @@ export default class ExploreLayout extends mixins(PrefixMixin) {
 }
 </script>
 
-<style scoped></style>
+<style scoped>
+.search-term {
+  font-size: 29px;
+}
+</style>
```
Comment (roiLeo) [inline components/rmrk/ExploreLayout/ExploreLayout.vue:61]: don't need this anymore with `is-size-3`
```suggestion
```

**Review by roiLeo — CHANGES_REQUESTED** [review]
- Mobile navbar is still breaking
Capture d’écran 2022-04-13 à 2 18 14 PM

**Comment by prachi00** [comment]
> * Mobile navbar is still breaking
>   Capture d’écran 2022-04-13 à 2 18 14 PM

yeah fixed now margin issue

**Comment by roiLeo** [comment]
BTW [this responsive mixins](https://bulma.io/documentation/utilities/responsive-mixins/#named-mixins) might help you for media queries

**Comment by prachi00** [comment]
> BTW [this responsive mixins](https://bulma.io/documentation/utilities/responsive-mixins/#named-mixins) might help you for media queries

will check this thank you

**Comment by prachi00** [comment]
> 😕 Capture d’écran 2022-04-13 à 2 43 47 PM

isnt this how it was before? I just fixed the margins

**Review by roiLeo — APPROVED** [review]
lgtm

**Comment by prachi00** [comment]
> 😕 Capture d’écran 2022-04-13 à 2 43 47 PM

this is because the branch isn't merged with the latest `main` branch, should be fine once merged

**Comment by roiLeo** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 152.44 USD/KSM ~ 0.328 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xaaf44207522aa789578d1918d421636d935e89834bba3b9c1ad516f9ba87ac80](https://kusama.subscan.io/extrinsic/0xaaf44207522aa789578d1918d421636d935e89834bba3b9c1ad516f9ba87ac80)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by prachi00** [comment]
> > this is because the branch isn't merged with the latest `main` branch, should be fine once merged
> 
> I think it's because I ask you to remove `card` class and I didn't know this class has a background color...

umm I just merged it with the latest branch, and it started to show up fine

**Comment by roiLeo** [comment]
> umm I just merged it with the latest branch, and it started to show up fine

let me know if it appair again

**Pull request merged**